source/header_cell_view/header_cell_view.tsx

```diff
@@ -90,31 +82,31 @@ export class HeaderCellView extends React.Component<Properties, State> {
       this.setState({isCondensed: true});
```

Comment:
This results in a race condition, refer to the ```updater``` in:

https://reactjs.org/docs/react-component.html#setstate

And return ```null``` to prevent an update.